Directory.Build.props

```diff
@@ -1,7 +1,14 @@
-﻿<Project>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
+  <PropertyGroup Condition="'$(CopyrightNetFoundation)' != ''">
+    <Copyright>$(CopyrightNetFoundation)</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Does this override `PackageLicenseUrl` if it's present? Or do we need to condition that on the inverse of this condition?

https://github.com/Microsoft/msbuild/blob/f42e4b11fb675b8164a5c666cfc2d42c5755bffd/src/Directory.Build.props#L59

Reply:
We need the condition on PackageLicenseUrl. Will add.